Tests/ArcGISToolkitTests/PreplannedMapModelTests.swift

```diff
@@ -136,7 +136,7 @@ class PreplannedMapModelTests: XCTestCase {
     
     @MainActor
     func testPackagingNotCompletePackageFailureStatus() async throws {
-        class MockPreplannedMapArea: PreplannedMapAreaProtocol {
+        final class MockPreplannedMapArea: PreplannedMapAreaProtocol, @unchecked Sendable {
             var packagingStatus: PreplannedMapArea.PackagingStatus? = nil
```

Comment:
Here, there's a simpler solution. Just make `packagingStatus` computed:

```suggestion
        final class MockPreplannedMapArea: PreplannedMapAreaProtocol {
            var packagingStatus: PreplannedMapArea.PackagingStatus? { nil }
```